api/net/Program.cs

```diff
@@ -238,6 +239,9 @@
 app.UseRouting();
 app.UseCors();
 
+app.UseMetricServer();
```

Comment:
Appears to be missing configuration required to connect to Prometheus.  Also according to documentation it should be configured before error handling middleware.

Reply:
Best practices require these endpoints to require authentication/authorization.  Adding it this way will be open to all.